package.json

```diff
@@ -21,6 +21,8 @@
     "format:imports": "biome check --formatter-enabled=false --write",
     "format:imports:ci": "biome ci --formatter-enabled=false",
     "test": "turbo run test --concurrency=1 --filter=astro --filter=create-astro --filter=\"@astrojs/*\"",
```

Comment:
Should we update this command to run `test:astro` and `test:integrations` now?

Reply:
Done in ec0a82af400188aa4056569783c8c5b09672df2b 👍